Dockerfile

```diff
@@ -6,8 +6,6 @@ WORKDIR /usr/src/app/
 COPY engine.json /
 COPY package.json /usr/src/app/
 
-ENV NODE_ENV production
```

Comment:
Ahh, I see. CircleCI has an inferred `npm install` step. Can we override that step to be a no-op?